libraries/rush-lib/src/logic/installManager/InstallHelpers.ts

```diff
@@ -71,6 +72,18 @@ export class InstallHelpers {
       }
 
       if (pnpmOptions.globalIgnoredOptionalDependencies) {
+        if (rushConfiguration.rushConfigurationJson.pnpmVersion !== undefined && semver.lt(rushConfiguration.rushConfigurationJson.pnpmVersion, '9.0.0')) {
+          // eslint-disable-next-line no-console
+          console.warn(
```

Comment:
Use `terminal`.

Reply:
The thing is that there's no `terminal` accessible in this scope. Should I rework the code a bit so it's passed down and accessible here or create a new instance of it?

Reply:
If that's easy enough to do, go for it, otherwise don't worry about it.